plugins/python/cloudpickle.py

```diff
@@ -1054,7 +1054,10 @@ def _fill_function(*args):
     #   content of state['globals'] to populate the shared isolated
     #   namespace with all the global variables that are specifically
     #   referenced for this function.
-    func.__globals__.update(state['globals'])
+    #   update only keys that does not exists in the current global dictionary
+    for k,v in state['globals'].items():
+        if k not in func.__globals__.keys():
```

Comment:
Can you use func.__globals__,update(state['globals']) here?

Reply:
This is how it was before, but updates override collides keys and this is not what I want.

Reply:
Correct - it would override all - that's why I was askling if it's different. Cool. approved.